django/core/checks/security/base.py

```diff
@@ -184,7 +191,7 @@ def check_ssl_redirect(app_configs, **kwargs):
 
 
 @register(Tags.security, deploy=True)
-def check_secret_key(app_configs, **kwargs):
+def check_secret_key_length(app_configs, **kwargs):
```

Comment:
Are the check method names considered part of the public API? Can they be changed or will it break backwards compat? This function has been in Django since 1.8.